shared-data/labware/schemas/3.json

```
@@ -67,8 +67,9 @@
     },
     "SphericalSegment": {
       "type": "object",
+      "description": "A list of all of the sections of the well that have a contiguous shape",
       "additionalProperties": false,
-      "required": ["shape", "radiusOfCurvature", "depth"],
+      "required": ["shape", "radiusOfCurvature", "topHeight"],
```

`bottomHeight` was intended to be required, right?

```suggestion
      "required": ["shape", "radiusOfCurvature", "bottomHeight", "topHeight"],
```